data/release/xdg/config.toml

```
@@ -35,7 +35,7 @@ enabled = true
 # Backend settings for communicating with the external API.
 [backend]
 # The endpoint points to an API server.
-endpoint = "https://console.redhat.com/api/lightspeed"
+endpoint = "https://cert.console.redhat.com/api/lightspeed"
```

without the /v1 it doesn't work for stage
```suggestion
endpoint = "https://cert.console.redhat.com/api/lightspeed/v1"
```